tasks/pocket_sand.py

```diff
@@ -0,0 +1,33 @@
+import os
+import random
+from discord_webhook import DiscordWebhook, DiscordEmbed
+import asyncio
+from tasks.base import BaseTask
+
+
+class DaleAttack(BaseTask):
+    """
+
+    When running, Dale Gibble randomly attacks with the person on thier turn, player or not
+
+    :param: None
+    :return: Embeded Discord Webhook
+    """
+
+    id = 'dale-attack'
+    channel = None
+
+    async def task(self):
+        webhook = DiscordWebhook(
+            url='https://discordapp.com/api/webhooks/596806839136419859/QtKJScaEKMfaTs2SMJVpBtuR610RtQVIUtunfgzzq3QBPw91-bLfry37lgKAlC710svS')
+        pocket_sand_url = 'https://i.kym-cdn.com/photos/images/original/001/295/678/f81.png'
+
+        if not self.channel:
+            self.channel = self.client.get_channel(os.getenv('POCKET_SAND_TOKEN'))
```

Comment:
What is POCKET_SAND_TOKEN? If it's supposed to be set from an environment variable, add it to the .template.env file

Reply:
Oh I see @michaelhillebrand's comment about it here: https://github.com/michaelhillebrand/huskiebot/pull/12#discussion_r300837774

It should still be added to the .template.env file